components/miro_custom_app/actions/update-sticky-note/update-sticky-note.mjs

```diff
@@ -0,0 +1,48 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  name: "Update sticky note",
+  version: "0.0.1",
+  key: "update-sticky-note",
+  description: "Updates content of an existing sticky note on a Miro board",
+  type: "action",
+  props: {
+    miro_custom_app: {
+      type: "app",
+      app: "miro_custom_app"
+    },
+    boardId: {
+      type: "string",
+      description: "Board ID",
+      optional: false,
+    },
+    itemId: {
+      type: "string",
+      description: "Shape (item) ID",
+      optional: false,
+    },
```

Comment:
Here you can create an `async options()` prop to get a dropdown menu to select both a `boardId` and an `itemId` using the api.